### PR TITLE
Deploy Spanish, CN,  (NOT Tagalog)

### PR DESCRIPTION
**Pull request by gregglind**

### CODE REVIEW

File: src/recipes/heartbeat-by-user-first-impression/index.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ let { hasAny } = require("../../jetpack/array");
 */
 
 const NAME="heartbeat by user v1";
-const VERSION=15;
+const VERSION=16;
```
Comment (willkg) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: This is a code refactoring to better handle l10n, but it shouldn't affect the outcome of the recipe. Should this version be increased?

File: src/localeStrings/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+module.exports = {
+  'de'   : require("./de"),
+  'en-us': require("./en"),  // break up 'en' if needful.
+  'en-gb': require("./en"),
+  'es':    require("./es"),  // should not exist
+  'es-es': require("./es"),  // break up 'es' if needful.
+  'es-mx': require("./es"),
+  'fr'   : require("./fr"),
+  'zh-cn': require("./zh-cn"),  // chose simplified chinese.  TW has traditional.
```
Comment (willkg) [inline src/localeStrings/index.js]: The PR comment says this adds Tagalog, but I don't see it listed here.

File: src/recipes/heartbeat-by-user-first-impression/index.js [hunk 3]
```diff
@@ -161,11 +150,11 @@ let run = function (state, extras) {
   eData.store();
 
   let locale = (state.locale || "UNK").toLowerCase();
-  let trans = translations[locale] || {};
+  let trans = translations[locale] || translations['en-us']; //
```
Comment (rayborn) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: Perhaps export a function in src/localeStrings/index.js instead?  That way we could handle default resolution there.

File: src/recipes/heartbeat-by-user-first-impression/index.js [hunk 4]
```diff
@@ -161,11 +150,11 @@ let run = function (state, extras) {
...
 
-  let question_text = trans.question_text || "Please rate Firefox";
-  let learnmore = trans.learnmore || "Learn more";
-  let thankyou = trans.thankyou || "Thank you!";
+  let question_text = trans.question_text();
```
Comment (rayborn) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: Don't see a test to guarantee that these exist.  Should we add a test or a "|| <default>"?

File: src/recipes/heartbeat-by-user-first-impression/index.js (hunk 3 above)
Comment (gregglind) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: done.  I like that idea.

File: src/recipes/heartbeat-by-user-first-impression/index.js (hunk 4 above)
Comment (gregglind) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: getTranslation will now *always* work.

File: src/recipes/heartbeat-by-user-first-impression/index.js (hunk 1 above)
Comment (gregglind) [inline src/recipes/heartbeat-by-user-first-impression/index.js]: Usually, I up it with any change at all the file, for debugging at input side.

File: src/localeStrings/index.js (hunk 2 above)
Comment (gregglind) [inline src/localeStrings/index.js]: Noted, will change comment.

**Pull request merged**